variables/ch3/Convert-birth-to-age.py

```diff
@@ -0,0 +1,15 @@
+# Use Python's built-in datetime module
+from datetime import date
+def age(birthdate):
+    today = date.today()
+    one_or_zero = ((today.month, today.day) < (birthdate.month, birthdate.day))
```

Comment:
how this line works !

Reply:
> how this line works !

It is related to the first or second six months of the year. That is, the variable "one_or_zero" will either be zero or one. If it becomes one, it subtracts one year from the person's age, and if it becomes zero, it does not subtract:
age = year_difference - one_or_zero